src/stdafx.h

```diff
@@ -81,29 +82,62 @@
 
 #include "zip.h"
 
+namespace fs = std::tr2::sys;
 #else
+#include "boost/filesystem.hpp"
+#include <stdarg.h>
+#include <unistd.h>
+namespace fs = boost::filesystem;
 
-//#include <unistd.h>
+#include <sys/syscall.h>
```

Comment:
This is Linux-specific. We'll probably want OS X and others, too, so let's use something more portable.

Actually, looking at where it's used, on Win32 it's only needed to post a message into the message queue of the thread. It is likely that any equivalent code on Linux and elsewhere will not use a thread ID for that purpose. So for now, I would suggest removing this, and just ifdef'ing all uses of `main_thread_id` on non-Win32. We'll figure out the proper abstraction for this later.